.github/workflows/Create-Release-Tags.yml

```
@@ -0,0 +1,26 @@
+name: Create release tags
+on:
+  workflow_dispatch: # Allow manual trigger
+    inputs:
+      agent-ref:
+        description: 'The ref (release tag) to create init containers for'
+        required: false
+        default: 'main'
```

We shouldn't need to check out the agent itself for this. We just need a version number. 
```suggestion
      agent-version:
        description: 'The release tag to create init containers for, including the v. Eg v8.12.0’. '
        required: true
```